gravitee-am-service/src/main/java/io/gravitee/am/service/model/openid/PatchSecurityProfileSettings.java

```diff
@@ -28,7 +26,9 @@
  */
 public class PatchSecurityProfileSettings {
 
-    public PatchSecurityProfileSettings() {}
+    public PatchSecurityProfileSettings() {
+        // empty
+    }
```

Comment:
suggestion: excepted if there is a valid reason to keep the constructor due to Jackson serialization or something lke this, maybe we can just remove the default constructor ? (same comment for all the `PatchxxxSettings` changes)

Reply:
fixed